### PR TITLE
chore(cypress): update token and documentation

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: test/integration/content-sidebar/TaskForm.e2e.test.js [hunk 1]
```diff
@@ -6,11 +6,10 @@ describe('Create Task', () => {
     const getMessageField = () => cy.getByTestId('task-form-name-input');
     const getSubmitButton = () => cy.getByTestId('task-form-submit-button');
     const getCancelButton = () => cy.getByTestId('task-form-cancel-button');
-    const username = 'Platform '; // will be used as assignee
+    const username = 'PreviewTestApp'; // will be used as assignee
 
     beforeEach(() => {
         cy.visit('/Elements/ContentSidebar'); // Open sidebar example page
-        cy.getByTestId('sidebaractivity').click(); // Open activity tab
```
Comment (jeremypress) [inline test/integration/content-sidebar/TaskForm.e2e.test.js]: We're not using a skills file by default so the sidebar will start on the activity feed.

File: conf/styleguide.setup.js [hunk 2]
```diff
@@ -2,9 +2,10 @@ import { IntlProvider } from 'react-intl';
 import features from '../examples/src/features';
 // Constants
 global.FEATURES = global.FEATURES || features;
-global.FILE_ID = global.FILE_ID || __FILEID__ || '308566420378'; // eslint-disable-line
-global.FOLDER_ID = global.FOLDER_ID || __FOLDERID__ || '51964781421'; // eslint-disable-line
-global.TOKEN = global.TOKEN || __TOKEN__ || 'aiMADZorjZDCJEfi7zREbvHBo2K70MXf'; // eslint-disable-line
+global.FILE_ID = global.FILE_ID || __FILEID__ || '415542803939'; // eslint-disable-line
+global.FOLDER_ID = global.FOLDER_ID || __FOLDERID__ || '69083462919'; // eslint-disable-line
+// NOTE: The token used is a readonly token accessing public data in a demo enterprise.
+global.TOKEN = global.TOKEN || __TOKEN__ || 'S8wjvjOL9GEK5VtXsQNVMOwSrx1g55oC'; // eslint-disable-line
```
Comment (priyajeet) [inline conf/styleguide.setup.js]: 🚨

File: test/integration/content-sidebar/TaskForm.e2e.test.js (hunk 1 above)
Comment (priyajeet) [reply test/integration/content-sidebar/TaskForm.e2e.test.js]: Dont want to add some skills? :o

**Comment by priyajeet** [comment]
Also in conf/styleguide.setup.js

File: test/integration/content-sidebar/TaskForm.e2e.test.js (hunk 1 above)
Comment (jeremypress) [reply test/integration/content-sidebar/TaskForm.e2e.test.js]: Added skills to all videos on the folder, but our base test file is a document.

**Pull request merged**